### PR TITLE
REGMON plugin functionality extended. Added support for Open and Create Registry functions.

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

File: src/plugins/regmon/regmon.h [hunk 1]
```diff
@@ -111,9 +111,9 @@
 class regmon: public plugin
 {
 public:
-    drakvuf_trap_t traps[3] =
+    drakvuf_trap_t traps[32] =
```
Comment (tklengyel) [inline src/plugins/regmon/regmon.h]: I don't think we need to allocate 32 traps here, it looks like you are using only 13 of them.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

File: src/plugins/regmon/regmon.h (hunk 1 above)
Comment (disaykin) [reply src/plugins/regmon/regmon.h]: Done.

**Comment by tklengyel** [comment]
Thanks, this looks good to me!

**Comment by disaykin** [comment]
@tklengyel, thanks!

**Comment by aoshiken** [comment]
@disaykin nice job

**Pull request merged**